oe_content.module

```diff
@@ -106,3 +109,128 @@ function oe_content_entity_base_field_info(EntityTypeInterface $entity_type) {
 function oe_content_locale_translation_projects_alter(&$projects) {
   $projects['oe_content']['info']['interface translation server pattern'] = drupal_get_path('module', 'oe_content') . '/translations/%project-%language.po';
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Add the composite field option to entity reference fields
+ */
+function oe_content_form_field_config_edit_form_alter(&$form, FormStateInterface $form_state): void {
+  /** @var \Drupal\field\FieldConfigInterface $field_config */
+  $field_config = $form_state->getFormObject()->getEntity();
+  $applicable_field_types = ['entity_reference', 'entity_reference_revisions'];
+  if (!in_array($field_config->getType(), $applicable_field_types)) {
+    return;
+  }
+  $form['composite'] = [
+    '#type' => 'radios',
+    '#title' => t('Composite field'),
+    '#description' => t('Making this field a composite field will delete any referenced entity if the referencing entity is deleted (only if the referenced entity is not referenced by some other entity).'),
+    '#default_value' => $field_config->getThirdPartySetting('oe_content', 'composite') ? 1 : 0,
+    '#options' => [
+      1 => t('Enabled'),
```

Comment:
Why we not making this a boolean? As the ticket says: `A simple boolean will be enough`

Reply:
If make the options boolean then for some reason they don't get checked properly once they are rendered.

Reply:
What?